File: dixit-companion/src/main/java/fr/erban/dxitcompanion/game/turn/Turn.java
package fr.erban.dxitcompanion.game.turn;

import java.io.Serializable;
import java.util.List;

import fr.erban.dxitcompanion.game.player.PlayerBean;
import fr.erban.dxitcompanion.game.turn.bean.VoteBean;
import lombok.Builder;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
@Builder
public class Turn implements Serializable {

    private boolean noOneFound;

    private boolean everybodyFound;

    private PlayerBean storyTeller;

    private List<VoteBean> votes;
}

File: dixit-companion/src/main/java/fr/erban/dxitcompanion/game/turn/Turn.kt
package fr.erban.dxitcompanion.game.turn

import android.os.Parcelable
import fr.erban.dxitcompanion.game.player.PlayerBean
import fr.erban.dxitcompanion.game.turn.bean.VoteBean
import kotlinx.parcelize.Parcelize

@Parcelize
data class Turn(
    val storyTeller: PlayerBean? = null,
    val votes: List<VoteBean> = emptyList(),
    val noOneFound: Boolean = false,
    val everybodyFound: Boolean = false
) : Parcelable
